_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why is it so important to do a pre-GSoC analysis
+and design it as early as possible, for it to be finalized in the cEP?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the **coala** community
+to discuss the new *major features* that the contributor intends to implement
+during the GSoC duration. The proposal is designed before the implementation of
+*major features* because the implementation usually happens from a technical
+perspective which may lead to a poorly designed content - developer and user-wise
+, so to avoid this, a cEP is designed at an earlier stage that can essentially
+save a lot of coding work and also makes sure that major features are properly
```

Comment:
`.. is designed at an earlier stage that can essentially save a lot of coding work ..` should be replaced by either `.. should be designed at an earlier stage and this essentially saves a lot of coding work ..` or `.. should be designed at an earlier stage so that it can essentially save a lot of coding work ..`

Reply:
Replacing with your 2nd suggestion

Reply:
@KVGarg I just edited that out, it wasn't grammatically sound. Use the first one, please.

Reply:
will`... should be designed at an earlier stage which essentially saves a lot of coding work ...` look good ?

Reply:
`as this` would be better

Reply:
`as this` would be better than `which` here